.github/workflows/ci.yaml

```
@@ -1349,7 +1349,7 @@ jobs:
       - name: Run CI workflow generation checks
         run: |
           ./ci/actions-templates/gen-workflows.sh
-          git diff --quiet
+          git diff
```

```suggestion
          git diff --exit-code
```
Sorry for the intrusion. `--quiet` [implies `--exit code`](https://git-scm.com/docs/git-diff). Plain `git diff` will exit with 0 even if there are differences.

---

@smoelius Thanks a lot!

---

@smoelius no intrusion at all, always happy to get your feedback!